src/adapters/mqtt/index.ts

```diff
@@ -205,8 +209,7 @@ class MqttAdapter extends Adapter {
 
   _send(message: GleeMessage): Promise<void> {
     return new Promise((resolve, reject) => {
-      const operation = this.parsedAsyncAPI.channel(message.channel).subscribe()
-      const binding = operation ? operation.binding('mqtt') : undefined
+      const binding = this.parsedAsyncAPI.channels().get(message.channel).bindings().get('mqtt').value()
```

Comment:
it gives :
```

x Cannot read properties of undefined (reading 'value')
    at file:///Users/khudadad/Projects/test/glee/dist/adapters/mqtt/index.js:167:103
```
for this line in `dummy` example.

Reply:
This is fixed now right, because the `dummy` example is working fine now.